fixity/reporting.py

```diff
@@ -1,16 +1,26 @@
 import calendar
 import json
+from datetime import datetime
+from typing import Optional
 
 import requests
 
+from .models import Report
+from .models import Session
 from .utils import check_valid_uuid
 
 
 class ReportServiceException(Exception):
     pass
 
 
-def post_pre_scan_report(aip, start_time, report_url, report_auth=(), session_id=None):
+def post_pre_scan_report(
+    aip: str,
+    start_time: Optional[datetime],
+    report_url: Optional[str],
```

Comment:
Why are these `Optional`?

Reply:
Same answer as above.

Reply:
Same argument as above :grin:

Reply:
Good one. :) I got your point. Let me adjust the tests.